### PR TITLE
Made log out button on dashboard log user out

### LINKED ISSUE #99: Logout
Add a logout link to the user menu that links to the logout endpoint.

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #99

#### What's this PR do?

change the log out button on the dashboard to, you know, log out. we just need to set the `href` prop on the `MenuItem` react bootstrap component.

#### Where should the reviewer start?

`static/js/containers/LoginButton.js`

#### How should this be manually tested?

make sure it logs out!

### CODE REVIEW

**Comment by noisecapella** [comment]
Here are our review guidelines, mostly general rules of thumb: https://github.com/mitodl/handbook/blob/master/code_review.md

Thanks for the PR! I'm going to make some comments but you got up and running really quickly which is great! Can you make these changes:
 - the git commit and PR title should be in the past tense. We have a script which generates our release notes from these PR titles
 - You should use `Fixes #99` instead of just `#99` so that github will close the connected issue when the PR is merged
 - Feel free to create a PR the minute you start working on an issue. When it is ready for review add the 'Needs Review' tag. After one of us reviews it we will change it to 'Waiting on Author', then the ball is back in your court to make changes. When you want another review change it back to 'Needs Review'

Other general rules:
 - We should take all comments on code reviews seriously and attempt to address them, but if you have a disagreement feel free to make a comment explaining why rather than making the suggested change.
 - When the reviewer comments with a :+1: it is up to you to merge
 - If you have substantial changes in response to a code review please make separate commits so that it's easy to tell which changes were made in response to the code review
 - When the code is ready to be merged the commits should be squashed into a single commit

Thanks again!

File: static/js/containers/LoginButton.js [hunk 1]
```diff
@@ -7,9 +7,23 @@ class LoginButton extends React.Component {
...
     const { authentication } = this.props;
 
-    return <DropdownButton title={authentication.name}>
-      <MenuItem eventKey="logout">Logout</MenuItem>
-    </DropdownButton>;
+    return (
+      <DropdownButton
+        title={authentication.name}
+        id="logout-button">
+        {this.logOutButton()}
```
Comment (noisecapella) [inline static/js/containers/LoginButton.js]: I'm not sure if it's worth making it a separate function here. Would it be possible to inline it?

Comment (aliceriot) [inline static/js/containers/LoginButton.js]: totally possible, I tend to write a lot of small helper methods with React, but this isn't a super complicated component so it's probably fine to just have one big `render` method.

**Comment by aliceriot** [comment]
@noisecapella cool, thanks for the info!

I updated the commit title. I have `closes #99` in the commit message itself - I didn't realize that we used PR templates. I think it just needs to be in the commit message (and not also the pr text) but I'm not sure...

**Comment by noisecapella** [comment]
You may be right about the commit message, feel free to leave it there

**Comment by noisecapella** [comment]
Everything looks good with functionality. Can you rename the PR title to the past tense and have it start with a capital letter, so it can go in the release notes unmodified?

After that, squash your commits and merge! :+1:

**Comment by aliceriot** [comment]
wooo! first pr!

**Pull request merged**